test/Benchmarks/Trace/TraceBenchmarks.cs

```diff
@@ -80,6 +82,21 @@ public TraceBenchmarks()
                 .AddProcessor(new DummyActivityProcessor())
                 .AddProcessor(new DummyActivityProcessor())
                 .Build();
+
+            Sdk.CreateTracerProviderBuilder()
+                .SetSampler(new AlwaysOnSampler())
+                .AddSource(this.sourceWithOneInstrumentation.Name)
+                .AddAspNetCoreInstrumentation()
```

Comment:
This benchmark has nothing to do with `AddAspNetCoreInstrumentation`. You can simply use the raw API directly - AddLegacyActivity("somename")

Reply:
I have removed the instrumentations and used AddLegacyActivity.